po/ja.po

```diff
@@ -390,7 +391,7 @@ msgstr "Avahi リゾルバが失敗しました: %s"
 #: ../avahi-ui/avahi-ui.c:518
 #, c-format
 msgid "Browsing for service type %s in domain %s failed: %s"
-msgstr "ドメイン %2$s にてサービスタイプ %1$s の検索に失敗しました: %3$s"
+msgstr "ドメイン %s にてサービスタイプ %s の検索に失敗しました: %s"
```

Comment:
As far as I can see `%1$` refers to `type`, `%2$` refers to `domain` and `%3$` refers to the result of `avahi_strerror` there so according to Google Translate with those specifiers the sentence translates into
```
Failed to look up service type TYPE in domain DOMAIN: ERROR
```
Without those specifiers it would be
```
Failed to look up service type DOMAIN in domain TYPE: ERROR
```

I don't speak Japanese and I don't know how accurate Google Translate is here though. @simmon-nplob I wonder how safe it is to drop the "%n$" specifiers and change the order here?

Reply:
Dear @evverx. This is a that reflects the terminology when translating Korean, and Japanese was also modified and applied accordingly with following a similar syntax.

Reply:
I'll put it back to its previous state for japan.

Reply:
> Since the grammatical structure is different, I think it would be better if it was stated in the original sentence

I agree that the fact that the `%n$` specifiers affect where `type`, `domain` and `error` are inserted could be highlighted better. I'm not sure how to do that in po files though.

Reply:
Thanks..@evverx, @rffontenelle ! Have a great time^^